extras/static_code_analysis.sh

```diff
@@ -109,6 +109,14 @@ if $RUNS_ON_TRAVIS; then
   fi
 fi
 
+
+export NEST_SOURCE=$PWD
+print_msg "MSGBLD0100: " "Running check for copyright headers"
```

Comment:
I don't remember the point of the numbering in the end. Is it correct that all messages here end with a `0100`?

Reply:
The basic idea of message numbers is the decoupling of the message text from the log parser, e.g., correcting a typo in a message may cause the log parser `extras/parse_travis_log.py` to fail. There is of course still a logical coupling by the message numbers, but this is much easier to handle than arbitrary text.

MSGBLD0100 is not a good choice as it is already used for retrieving the list of changed files.I would suggest using MSGBLD1x00 while x should be chosen so that some numbers between 1040 - 1x00 are reserved. This simplifies possible later extensions of the script.